### ISSUE TITLE
New FileAttribute UI component -- what is a validMetisPath?

### ISSUE THREAD

**Issue by coleshaw** [mountetna/timur#251]
In the new `file_attribute.jsx` component, there is reference to a [validMetisPath() method](https://github.com/mountetna/timur/blob/graft-integrate-metis/lib/client/jsx/components/attributes/file_attribute.jsx#L84). What constitutes a valid Metis path? The method is not defined in the component.

Thanks!

**Comment by graft** [comment]
I should clarify the completion state of these various branches. It reaches across Magma, Metis and Timur and is hard to follow. We should write up a road map for this work and put it in mountetna/magma#51.  We can convert this issue to discussing the remaining FileAttribute work in Timur.

In this case I think the answer is I simply had not written this yet and the "valid metis path" was not defined yet. I have started some work (which also needs writing up and completion) on a [metis client](https://github.com/mountetna/metis/tree/graft-metis-client); therein I define a format for a "metis path":

https://github.com/mountetna/metis/blob/5cc852b3fda16e257b2dd9bc5df4fd24b67d6671/bin/metis_client#L181-L191

It's not clear whether this is a good format yet, nor is it clear how to easily share this definition across codebases and languages.
